static/app/components/events/aiAutofix/autofixCard.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled';
+
+import {FixResult} from 'sentry/components/events/aiAutofix/fixResult';
+import {Step} from 'sentry/components/events/aiAutofix/step';
+import type {AutofixData} from 'sentry/components/events/aiAutofix/types';
+import {ExperimentalFeatureBadge} from 'sentry/components/events/aiSuggestedSolution/experimentalFeatureBadge';
+import Panel from 'sentry/components/panels/panel';
+import PanelBody from 'sentry/components/panels/panelBody';
+import PanelFooter from 'sentry/components/panels/panelFooter';
+import PanelHeader from 'sentry/components/panels/panelHeader';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+export function AutofixCard({data, onRetry}: {data: AutofixData; onRetry: () => void}) {
```

Comment:
minor nit: Typically would pull props into like a `interface Props {}` thing. not important